docs/baseband/tutorials/getting_started.rst

```diff
@@ -0,0 +1,317 @@
+    0
+    >>> d_complete = fh.read()
+    >>> d_complete.shape
+    (40000, 8)
+
+.. note::
+
+    ``fh.read()`` returns a **copy** of the data from ``fh``.
+
+We can also move the pointer with respect to the end of file by passing ``2``
+as a second argument (as with `io.BufferedReader` pointers)::
+
+    >>> fh.seek(-100, 2)    # Second arg is 0 (start of file) by default.
+    39900
+    >>> d_end = fh.read(100)
+    >>> np.array_equal(d_complete[-100:], d_end)
+    True
+
+``-100`` means 100 counts before the end of file, so ``d_end`` is equal to
+the last 100 entries of ``d_complete``.  Baseband only keeps the most recently
+accessed data frame in memory, so selective decoding using ``seek`` and
+``read`` is useful when examining extremely large files.
+
+To determine where the pointer is located, we use ``fh.tell()``::
+
+    >>> fh.tell()
+    40000
+    >>> fh.close()
+
+Seeking and Telling in Time With the Sample Pointer
+---------------------------------------------------
+
+We can use ``seek`` and ``tell`` with units of time rather than samples.  To do
+this with ``tell``, we can pass an appropriate `astropy.units.Unit` object to
+its optional ``unit`` parameter::
+
+    >>> fh = vdif.open(SAMPLE_VDIF, 'rs')
+    >>> fh.seek(40000)
+    40000
+    >>> fh.tell(unit=u.ms)
+    <Quantity 1.25 ms>
+
+Passing the string ``'time'`` reports the pointer's location in absolute time::
+
+    >>> fh.tell(unit='time')
+    <Time object: scale='utc' format='isot' value=2014-06-16T05:56:07.001250000>
+
+We can also pass an absolute `astropy.time.Time`, or a positive or negative time
+difference `~astropy.time.TimeDelta` or `astropy.units.Quantity` to ``seek``. 
+If the offset is a `~!astropy.time.Time` object, the second argument to seek is
+ignored.
+
+::
+
+    >>> from astropy.time.core import TimeDelta
+    >>> from astropy.time import Time
+    >>> fh.seek(TimeDelta(-5e-4, format='sec'), 2)  # Seek -0.5 ms from end.
+    24000
+    >>> fh.seek(0.25*u.ms, 1)  # Seek 0.25 ms from current position.
+    32000
+    >>> # Seek to time index 2014/06/16 5:56:07.001125
+    >>> fh.seek(Time('2014-06-16T05:56:07.001125'))
+    36000
+    >>> fh.close()
+
+Extracting Header Information
+-----------------------------
+
+The first header of the file is stored as the ``header0`` attribute of the
+stream reader object; it gives direct access to header properties via keyword
+lookup::
+
+    >>> with vdif.open(SAMPLE_VDIF, 'rs') as fh:
+    ...     header0 = fh.header0
+    >>> header0['frame_length']
+    629
+
+The full list of keywords is available by printing out ``header0.keys()``.
+
+A number of derived properties, such as the time (as a `~astropy.time.Time`
+object), are also available through the header object.  
+
+    >>> header0.time
+    <Time object: scale='utc' format='isot' value=2014-06-16T05:56:07.000000000>
+
+These are listed in the API under each header class's entry.  For example,
```

Comment:
Should we recommend to just tab-complete instead?

Reply:
I personally like APIs, and feel users will use tab-completion regardless of whether we recommend it.

Reply:
good point, though perhaps we should be more explicit in our classes about what is defined. The properties are partially copied from superclasses.

Reply:
Possibly, it should not be a private property...

Reply:
I think this is related to issue #84 that I raised, and I'll look into better documenting properties (though perhaps in the next PR).